snippets/ztp_example-sno.yaml

```diff
@@ -0,0 +1,81 @@
+# example-node1-bmh-secret & assisted-deployment-pull-secret need to be created under same namespace example-sno
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "example-sno"
+  namespace: "example-sno"
+spec:
+  baseDomain: "example.com"
+  cpuPartitioningMode: AllNodes
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "ssh-rsa AAAA..."
+  clusters:
+  - clusterName: "example-sno"
+    networkType: "OVNKubernetes"
+    installConfigOverrides:  "{\"capabilities\":{\"baselineCapabilitySet\": \"None\", \"additionalEnabledCapabilities\": [ \"marketplace\", \"NodeTuning\" ] }}"
```

Comment:
This is horrible for readability
Won't a multi-line yaml string work?

Reply:
Updated. Also tweaked `installConfigOverrides`.